document/chapters/02-background.tex

```diff
@@ -929,14 +929,109 @@ \subsection{Signed Distance Fields}
 is repeatedly clipped using signed distance values to approach a surface in a
 process called ray marching \cite{Tomczak_2012}.
 
+\section{Monte Carlo Memory Considerations}\label{sec:mc_mem}
+
+The bulk of Monte Carlo memory consumption in native calculations is attributed
+to nuclear cross-section data and tally data structures with the analytic CSG
+geometry representation taking up a small portion of the overall program memory.
+
+\subsection{Cross-Section Data}
+
+Cross-section data is necessary to represent the physical
+processes governing particle transport through the virtual geometry. Various
+formats of this data exist with both continuous and discrete representation of
+cross section information. Continuous representations of data are compact in
```

Comment:
I don't think this is true.  CE cross-sections are often more memory intensive than group cross-sections, as well as being more time consuming to evaluate.  They are more accurate, though!

Reply:
I should have done more reading on this, sorry. I'm sure you are correct, but for my benefit I'm going to read up on it a bit.